cmd/scaler/README.md

```diff
@@ -0,0 +1,36 @@
+# A cli tool to scale k8s deployments from within a k8s cluster. 
+
+This tool uses [k8s provider](../../pkg/provider/k8s) to scale a deployment up and down periodically, from within a k8s cluster.
+
+## Build
+The project uses [go modules](https://github.com/golang/go/wiki/Modules) so it requires go with support for modules.
+
+```
+go build scaler.go
+// reads go.mod from the project root and downloads all dependancies.
+```
+
+## Usage
+```
+// (Note: These commands should be executed inside a k8s container)
```

Comment:
should also mention an example of how to set the corret rbac permission

kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: loadgen-scaler
  namespace: prombench-{{ .PR_NUMBER }}
rules:
- apiGroups: ["apps"]
  resources:
  - deployments
  verbs: ["get", "list", "update"]
---